cowprotocol/solver_dashboard/batch_settlement_info_1373033.sql

```diff
@@ -0,0 +1,53 @@
+-- The fee value and cost coverage is no longer used in the current mechanism
+-- Parameters
+-- {{blockchain}} - blockchain name
+
+WITH 
+num_tokens_traded as (
+    select 
+        hash,
+        count(distinct(token)) num_tokens 
+    from (
+        select evt_tx_hash as hash, "buyToken" token from gnosis_protocol_v2_{{blockchain}}.GPv2Settlement_evt_Trade
+        UNION
+        select evt_tx_hash as hash, "sellToken" token from gnosis_protocol_v2_{{blockchain}}.GPv2Settlement_evt_Trade
+    ) as all_tokens
+    group by hash
+)
+
+SELECT 
+    block_time,
+    num_trades,
+    num_tokens,
+    dex_swaps,
+    batch_value,
+    gas_used / num_trades as gas_per_trade,
+    case when '{{blockchain}}'='ethereum' then
+        CONCAT('<a href="https://etherscan.io/address/', cast(solver_address as varchar), '" target="_blank">', concat(environment, '-', name),  '</a>') 
+        when '{{blockchain}}'='gnosis' then
+        CONCAT('<a href="https://gnosisscan.io/address/', cast(solver_address as varchar), '" target="_blank">', concat(environment, '-', name),  '</a>') 
+        when '{{blockchain}}'='arbitrum' then
+        CONCAT('<a href="https://arbiscan.io/address/', cast(solver_address as varchar), '" target="_blank">', concat(environment, '-', name),  '</a>') 
+        end as solver,
+    
+    case when '{{blockchain}}'='ethereum' then
+        CONCAT('<a href="https://etherscan.io/tx/', cast(tx_hash as varchar), '" target="_blank">', cast(tx_hash as varchar),  '</a>') 
+        when '{{blockchain}}'='gnosis' then
+        CONCAT('<a href="https://gnosisscan.io/tx/', cast(tx_hash as varchar), '" target="_blank">', cast(tx_hash as varchar),  '</a>') 
+        when '{{blockchain}}'='arbitrum' then
+        CONCAT('<a href="https://arbiscan.io/tx/', cast(tx_hash as varchar), '" target="_blank">', cast(tx_hash as varchar),  '</a>') 
+    end as txHash,
+    gas_price / pow(10, 9) as gas_price,
```

Comment:
Checked the current version of the query on Dune (not sure if it is identical to this PR though) and gas price looks like it's a integer-> https://dune.com/queries/1373033
so some fixing is probably needed here

Reply:
This comes from the way it is being displayed.
In the data it is represented as the value of Gwei for one unit of gas which is not an integer. But then, the table has been rounded to the closest integer, in the display parametets.